tests/test_xyz/test_points.py

```diff
@@ -165,11 +165,13 @@ def test_export_points_rmsattr(testpath, tmp_path):
         testpath / POINTSET4
     )  # should guess based on extesion
     output_path = tmp_path / "poi_export.rmsattr"
-    print(mypoints.dataframe)
 
     mypoints.to_file(output_path, fformat="rms_attr")
     mypoints2 = xtgeo.points_from_file(output_path)
-    print(mypoints2.dataframe)
 
     assert mypoints.dataframe["Seg"].equals(mypoints2.dataframe["Seg"])
-    assert mypoints.dataframe["MyNum"].equals(mypoints2.dataframe["MyNum"])
+
+    np.testing.assert_array_almost_equal(
```

Comment:
Probably want to squash 79e37e59898f36 and bcf7b2a1e1 as the first one only introduces print statements but the other changes it to an assert.